parser/prism/Translator.h

```diff
@@ -63,13 +69,16 @@ class Translator final {
     std::unique_ptr<SorbetAssignmentNode> translateOpAssignment(pm_node_t *node);
 
     template <typename PrismLhsNode, typename SorbetLHSNode>
-    std::unique_ptr<SorbetLHSNode> translateConst(PrismLhsNode *node);
+    std::unique_ptr<parser::Node> translateConst(PrismLhsNode *node);
```

Comment:
I had to be less specific about the return type, to account for the case where if `SorbetLHSNode` is `parser::CVarLHS`, where we need to return a `parser::LVarLhs` for the workaround instead.